pkg/resources/function.go

```diff
@@ -76,7 +83,7 @@ var functionSchema = map[string]*schema.Schema{
 		Type:         schema.TypeString,
 		Optional:     true,
 		ForceNew:     true,
-		ValidateFunc: validation.StringInSlice(languages, false),
+		ValidateFunc: validation.StringInSlice(languages, true),
```

Comment:
the same remark as above

Reply:
there is no behavior change, since we were already ignoring the case diff. Even the default value of languages is "SQL", which is all caps, yet the languages array has it as lower case. But if you want i can add this to the migration guide.